website/client/app.vue

```diff
@@ -321,12 +321,12 @@ export default {
     }
 
     // Manage modals
-    this.$root.$on('bv::show::modal', (modalId, data = {}) => {
-      if (data.fromRoot) return;
+    this.$root.$on('bv::show::modal', ({modalId, fromRoot = false, alreadyTracked = false}) => {
```

Comment:
isn't modalId passed as the first parameter when bootstrap-vue emits this event?